Salary App/Features/SalaryForm/View/SalaryBreakDownViewController.swift

```diff
@@ -0,0 +1,261 @@
+        stackView.alignment = .fill
+        stackView.distribution = .equalSpacing
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.spacing = 24
+        return stackView
+    }()
+    private lazy var originalSalaryLbl: UILabel = {
+        let originalSalaryLbl = UILabel()
+        originalSalaryLbl.translatesAutoresizingMaskIntoConstraints = false
+        return originalSalaryLbl
+    }()
+
+    
+    private lazy var salaryInfoVw: SalaryInfoView = {
+        let salaryInfoVw = SalaryInfoView()
+        return salaryInfoVw
+    }()
+    
+    private let salaryBreakdown: SalaryBreakdown
+    
+    init(salaryBreakdown: SalaryBreakdown) {
+        self.salaryBreakdown = salaryBreakdown
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    @objc func openLink() {
+        
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setup()
+    }
+}
+
+extension SalaryBreakDownViewController {
+    
+    func setup() {
+        view.backgroundColor = UIColor(named: "darkMode")
+        title = "My Salary Breakdown"
+        self.navigationController?.navigationBar.prefersLargeTitles = true
+        view.addSubview(stackView)
+        
+        
+        stackView.addArrangedSubview(originalStackView)
+        originalStackView.addArrangedSubview(originalSalaryLbl)
+
+        stackView.addArrangedSubview(monthlyStackView)
+        monthlyStackView.addArrangedSubview(salaryMonthlyLbl)
+
+        stackView.addArrangedSubview(dailyStackView)
+        dailyStackView.addArrangedSubview(salaryDailyLbl)
+        
+        
+        stackView.addArrangedSubview(hourlyStackView)
+        hourlyStackView.addArrangedSubview(salaryHourlyLbl)
+        
+        stackView.addArrangedSubview(weeklyStackView)
+        weeklyStackView.addArrangedSubview(salaryWeeklyLbl)
+        
+        stackView.addArrangedSubview(taxStackView)
+        taxStackView.addArrangedSubview(taxLbl)
+        
+        stackView.addArrangedSubview(niStackView)
+        niStackView.addArrangedSubview(niLbl)
+        
+        stackView.addArrangedSubview(studentLoanStackView)
+        studentLoanStackView.addArrangedSubview(studentLoanLbl)
+        
+        stackView.addArrangedSubview(salaryInfoVw)
+        
+        NSLayoutConstraint.activate([
+            stackView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor, constant: 32),
+            stackView.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor, constant: 16),
+            stackView.trailingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor, constant: -16)
+            
+        ])
+        
+        originalSalaryLbl.text = "Orignial salary: £\(salaryBreakdown.originalSalary)"
+
+        salaryMonthlyLbl.text = "Monthly Total: £\(salaryBreakdown.monthlySalary)"
```

Comment:
Rather than hard coding the `£` you have a number formatter for currency use this instead for all of these